fastlane/Fastfile

```diff
@@ -447,28 +447,53 @@ platform :ios do
   lane :generate_docs do
```

Comment:
let me know if the logic in this new version of the lane is hard to follow. 
it does: 
- doc generation
- clone the docs repo
- copy the stuff over to the cloned repo
- git add, commit, push